riscv_isa.cpp

```diff
@@ -1045,3 +1049,30 @@ void ac_behavior(FNMADD_S){
   save_float(res, rd);
   dbg_printf("Result = %.3f\n\n", res);
 }
+
+
+// Instruction FLD behavior method
+void ac_behavior(FLD){
+  int imm;
+  imm = (imm4 << 11) | (imm3 << 5) | (imm2 << 1) | imm1;
+  dbg_printf("FLD r%d, r%d, %d\n", rd, rs1, imm);
+  int sign_ext;
+  sign_ext = sign_extend(imm, 12);
+  RBF[rd] = DM.read(RB[rs1] + sign_ext);  
```

Comment:
A double is a 64-bit value, but here you load and save only half of it.
See https://github.com/ArchC/mips/blob/mips32r2/mips_isa.cpp#L100-L107 for an example of a code that loads a 64-bit value.